### PR TITLE
Alert api + postman tests

### PR DESCRIPTION
**Pull request by hk-dassana**

### CODE REVIEW

File: engine/src/main/java/app/dassana/core/restapi/Alert.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            s3Client
+                    .getObject(GetObjectRequest.builder().
+                            bucket(s3Bucket).key(uploadedPath)
+                            .build(), (getObjectResponse, abortableInputStream) -> { //handle the object
+                        String customWorkflowFile = IOUtils.toString(abortableInputStream, Charset.defaultCharset());
+                        alert.append(customWorkflowFile);
+                        return null;
+                    });
+        } catch (S3Exception exception) { // using the micronaut HttpStatusException to handle exceptions
+            System.out.println(exception.getMessage());
```
Comment (gauravphoenix) [inline engine/src/main/java/app/dassana/core/restapi/Alert.java:49]: please use logger and not system out

**Comment by gauravphoenix** [comment]
awesome to see postman tests, but please add unit tests also as they are run against every PR against main branch

File: engine/src/main/java/app/dassana/core/restapi/Alert.java (hunk 1 above)
Comment (hk-dassana) [reply engine/src/main/java/app/dassana/core/restapi/Alert.java]: this was for testing purposes, i have removed it from code.

**Review by gauravphoenix — DISMISSED** [review]

File: engine/src/main/java/app/dassana/core/restapi/Alert.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    @Inject
+    S3Client s3Client;
+
+    @Value("${env.dassanaBucket}")
+    String s3Bucket;
+
+    // The @Get annotation maps the index method to all requests that use an HTTP GET
+    @Get
+    public String getAlert(@QueryValue("vendorId") String vendorId, @QueryValue("alertId") String alertId) {
```
[the remark this answers is not in the document]
Comment (hk-dassana) [reply engine/src/main/java/app/dassana/core/restapi/Alert.java]: change made

**Review by gauravphoenix — APPROVED** [review]

**Pull request merged**